tools/hidl/hidlcomp.cpp

```diff
@@ -384,12 +387,22 @@ void indentOutf1(int inc, const char* fmt,...) __attribute__((format(printf,2,3)
 void indentOutf1(int inc, const char* fmt, ...)
 {
     indent(gIndent+inc);
-    
+
     va_list args;
     va_start(args, fmt);
     voutf(fmt,args);
 }
 
+void validateProfileExecutionOptions(std::string &options)
```

Comment:
For a string validation function, a comment showing the expected format would be helpful.